packages/vx-shape/build/index.js

```diff
@@ -67,6 +67,15 @@ Object.defineProperty(exports, 'LinkRadial', {
   }
 });
 
+var _Area = require('./shapes/Area');
```

Comment:
not sure how the build file got in, but looks like it's been there for a while. let me know if you want me to remove it here!

Reply:
🤦‍♂️ my fault. I'll fix this.